test/e2e/app-dir/layout-params/next.config.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   experimental: {
     appDir: true,
-    fontLoaders: [],
```

Comment:
Seems this file was moved instead of copied? 🤔

Reply:
:+1: